code/client/src/pages/Create.jsx

```diff
@@ -60,52 +76,72 @@ const Create = () => {
   const [progressStage, setProgressStage] = useState(0)
   const [address, setAddress] = useState() // '0x12345678901234567890'
   const [effectiveTime, setEffectiveTime] = useState()
+  const [doubleOtp, setDoubleOtp] = useState(false)
 
   const [durationVisible, setDurationVisible] = useState(false)
-  const [section, setSection] = useState(2)
+  const [section, setSection] = useState(sectionViews.setupOtp)
   const [qrCodeData, setQRCodeData] = useState()
   const [otp, setOtp] = useState('')
 
   const [deploying, setDeploying] = useState()
 
   const otpRef = useRef()
 
-  const getQRCodeUri = () => {
+  const getQRCodeUri = (otpSeed) => {
     // otpauth://TYPE/LABEL?PARAMETERS
-    return `otpauth://totp/${name}?secret=${b32.encode(seed)}&issuer=Harmony`
+    return `otpauth://totp/${name}?secret=${b32.encode(otpSeed)}&issuer=Harmony`
```

Comment:
Need to use a slightly different name here. e.g. instead of `${name}`, use `${name} (2nd)`

Reply:
So the `name` is the same, but append `(2nd)`?

Reply:
yes